resources/client/cl_main.ts

```diff
@@ -179,6 +179,14 @@ on(`__cfx_nui:${PhoneEvents.CLOSE_PHONE}`, async (data: any, cb: Function) => {
   cb();
 }); // Called for when the phone is closed via the UI.
 
+RegisterNuiCallbackType(PhoneEvents.TOGGLE_KEYS);
+on(`__cfx_nui:${PhoneEvents.TOGGLE_KEYS}`, async (data: any, cb: Function) => {
+  if (isPhoneOpen) {
+		SetNuiFocusKeepInput(data.keepGameFocus);
```

Comment:
Indentation here is a little odd.

Reply:
Seems to be a github issue because the indentation is good in my editor. Is probably because I use tabs instead of spaces but will change it to spaces hopefully that will fix the oddness
![image](https://user-images.githubusercontent.com/5462901/119322164-ac256d80-bc7d-11eb-8e3c-dc2ae52eceed.png)